person.rb

```diff
@@ -27,4 +28,9 @@ def can_use_services?
   def correct_name
     @name
   end
+
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self
+  end
```

Comment:
- [ ] Great job for adding the method but I think it would be great if you add the right association `qne achieving the 'has-many' association` here as well since this method it will be between the book and date in Person file so you need to pass a book and date as a parameter  then you can create a new Rental with Rental.new and pass to this new `(date, book, self)` 🤩✨💯🔥 you can check this [example](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) for more explaination